tests/integration/workflow/test_test_case.py

```diff
@@ -201,3 +201,8 @@ def test__edit__remove_only(
 
     assert test_case.version == 2
     assert sorted(test_case.load_test_samples()) == sorted(rest)
+
+
+def test__init__validate_name() -> None:
+    with pytest.raises(ValueError):
+        TestCase("")
```

Comment:
Should test `create` as well

Reply:
create is tested in tests/unit/workflow/test_test_case